README.md

```diff
@@ -48,3 +49,71 @@ docker-compose up -d matchbox
+- Error conditions are satisfied (if applicable for the transaction)
+
+## Transaction Details
+
+### DDCC:VS Generation Service
+
+#### Submit Health Event
+
+This transaction submits a Health Event Request consisting of a batch Bundle of [Parameters](https://worldhealthorganization.github.io/ddcc/StructureDefinition-DDCCGenerateHealthCertificateParameters.html) for the [Generate Health Certificate Operation](https://worldhealthorganization.github.io/ddcc/OperationDefinition-DDCC-QuestionnaireResponse-generateHealthCertificate.html). The transaction returns a Submit Health Event Response consisting of a batch-response FHIR Bundle. To learn more about this transaction, refer to the [Submit Health Event workflow in the WHO specification](https://worldhealthorganization.github.io/ddcc/transactions.html#submit-health-event).
+
+The following assertions are made for this transaction:
+
+- Valid Transaction: When a valid Health Event Request is sent via POST to `[base]/submitHealthEvent`, the response should:
+  - Return 201 status.
+  - Return a response body that maintains elements from the request such as the DDCCDocument HCID, DDCCPatient name and birthdate, and DDCCImmunization vaccine information.
+- Invalid Transaction (422 Error): When a Health Event Request is sent via POST to `[base]/submitHealthEvent` that does not conform to the [DDCCSubmitHealthEvent profile](http://worldhealthorganization.github.io/ddcc/StructureDefinition/DDCCSubmitHealthEventRequest), the response should return 422 status.
+
+#### Generate Health Certificate
+
+This FHIR operation accepts a set of immunization data elements via a [FHIR Parameters resource](https://worldhealthorganization.github.io/ddcc/StructureDefinition-DDCCGenerateHealthCertificateParameters.html) and returns [DDCCDocument resource](https://worldhealthorganization.github.io/ddcc/StructureDefinition-DDCCDocument.html) as the response.
+To learn more about this transaction, refer to the [Generate Health Certificate OperationDefinition](https://worldhealthorganization.github.io/ddcc/OperationDefinition-DDCC-QuestionnaireResponse-generateHealthCertificate.html).
+
+The following assertions are made for this transaction:
+
+- Valid Transaction: When a valid Generate Health Certificate Request is sent via POST to `[base]/QuestionnaireResponse/$generateHealthCertificate`, the response should:
+  - Return 201 status.
+  - Return a valid [DDCCDocument resource](https://worldhealthorganization.github.io/ddcc/StructureDefinition-DDCCDocument.html) as the response body.
+- Invalid Transaction (422 Error): When a Generate Health Certificate Request is sent via POST to `[base]/QuestionnaireResponse/$generateHealthCertificate` that does not conform to the [DCCCGenerateHealthCertificateParameters profile](https://worldhealthorganization.github.io/ddcc/StructureDefinition-DDCCGenerateHealthCertificateParameters.html), the response should return 422 status.
```

Comment:
Feel free to re-word, but we should mention something about expecting the response to have a signature, since this isn't necessarily required just by the profile validation.